context/src/examples/Contact.ts

```diff
@@ -0,0 +1,8 @@
+interface Contact extends Context {
```

Comment:
I am not sure where these examples should go, into a general examples folder from the root, or inside the src for context data. Leave them as a subfolder for now I guess.

Reply:
moved to example folder